ec/ecresource/deploymentresource/read.go

```diff
@@ -65,7 +65,11 @@ func (r *Resource) Read(ctx context.Context, request resource.ReadRequest, respo
 	}
 
 	// use state for the plan (there is no plan and config during Read) - otherwise we can get unempty plan output
-	newState, diags = r.read(ctx, curState.Id.ValueString(), &curState, nil, nil, privateFilters)
+	var migrateTemplateRequest *deployments.MigrateDeploymentTemplateOK
+	newState, diags = r.read(ctx, curState.Id.ValueString(), &curState, nil, nil, privateFilters, migrateTemplateRequest)
+
+	// Store migrate request in private state
+	updatePrivateStateMigrateTemplateRequest(ctx, response.Private, migrateTemplateRequest)
```

Comment:
Similar to other places - `migrateTemplateRequest` is `nil` here.

Reply:
Addressed in https://github.com/elastic/terraform-provider-ec/pull/755/commits/4c70d0034521bc6e2be803e5472948612fb64f2c.